ci-operator/config/redhat-developer/devconsole-operator/redhat-developer-devconsole-operator-master.yaml

```diff
@@ -25,6 +25,10 @@ images:
   dockerfile_path: openshift-ci/Dockerfile.deploy
   to: devconsole-operator
 
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: ci
```

Comment:
I don't think we want unrelated (I mean unrelated to the actual CI tooling) images to be promoted into the `ci` namespace... I think this should do what `knative` is doing and promote into an imagestream in `openshift`, or maybe have a NS of it's own, not sure.

@stevekuznetsov @smarterclayton any thoughts?

Reply:
Thanks for the feedback @petr-muller !

Yeah, knative does this:

```
cluster: https://api.ci.openshift.org
  namespace: openshift
  name: knative-v0.3
```

I could go with something like

```
cluster: https://api.ci.openshift.org
  namespace: openshift
  name: devconsole-0.1.0
```

Reply:
Done 
```
promotion:
  cluster: https://api.ci.openshift.org
  namespace: devconsole
  tag: latest
```

Reply:
https://github.com/openshift/release/pull/3357/commits/1429a01a5e54ab8ef04dd9efd0c2340a43958727

Reply:
Looks good. You'll probably need to add some minimal config (to set up that namespace) to https://github.com/openshift/release/tree/master/projects

Reply:
Thanks @petr-muller , will do.

Reply:
The README says the `projects` directory is deprecated.
https://github.com/openshift/release#component-project-build-manifests

Reply:
Is there a good reason to need a new namespace? Just use the first of your two suggestions and you need no extra config

Reply:
@stevekuznetsov You mean, this should do? 
```
cluster: https://api.ci.openshift.org
  namespace: openshift
  name: devconsole-0.1.0
```